js/index.js

```diff
@@ -1,7 +1,46 @@
 // Iteration 1: Names and Input
-
+let hacker1="Elia"
+console.log(`The driver's name is ${hacker1}`)
+let hacker2="ceyda"
+console.log(`The navigator's name is ${hacker2}`)
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length){
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+  } else if (hacker1.length<hacker2.length){
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length}  characters.`)
+  } else{
+  `Wow, you both have equally long names, ${hacker1.length}  characters!.`}
+  
+
+// Iteration 3: Loops
+for (let i=0; i<hacker1.length; i++){
+    console.log(hacker1.split('').join(' ').toUpperCase());
```

Comment:
Please be consistent in using semicolons. Either use them at every end of the line or don't.